config/initializers/wicked_pdf.rb

```diff
@@ -0,0 +1,5 @@
+require 'wicked_pdf'
```

Comment:
This shouldn't be needed but all tests would fail with `uninitialized constant WickedPdf` otherwise, and I'm clueless and it's late. :sleeping: